RELEASES.md

```diff
@@ -8,6 +8,11 @@
   Additionally, made the `library/preinstall` endpoint able to install any
   transitive dependencies of the library.
 
+## Interpreter/Runtime
+
+- Added support for overloaded conversions. This allows the `from` method to
+  be implemented with several overloads. ([#3227](https://github.com/enso-org/enso/pull/3227))
+
```

Comment:
If I understand correctly, we are aiming to have a single changelog and currently the one to use is at `app/gui/CHANGELOG.md`. cc: @mwu-tow please correct me if I'm wrong

Reply:
Happy to put this in the right place. I was just reacting to a red checkmark in the build process.

Reply:
Yes, `app/gui/CHANGELOG.md` is the location currently enforced by the CI. It'll be moved to the repository root,